acceptance/gossip_peerings_test.go

```diff
@@ -45,11 +44,6 @@ func checkGossip(t *testing.T, l *localcluster.Cluster, d time.Duration,
 		case <-stopper:
 			t.Fatalf("interrupted")
 			return nil
-		case e := <-l.Events:
-			if log.V(1) {
-				log.Infof("%+v", e)
-			}
-			return fmt.Errorf("event: %+v", e)
```

Comment:
I would always use AssertAndStop, but for tests that read from the events channel themselves AssertAndStop would trivially pass because the test had already drained the channel.

I want to avoid boilerplate in tests that have the default expectations. Reducing boilerplate in tests that have non-default expectations would be nice too once we have more than one such test, but it's lower priority (and ideally we'd make the default expectations smart enough that custom handling remains rare).